content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
 draft: false
 ---
 
-### Global site parameters
+In this guide, we'll be covering the following starting actions for wasmCloud development:

```

Comment:
The phrase "getting started" here I thought was a bit more figurative. Rather than telling people "how to start all the things", this is really just the first steps you need to take as a developer to get started on your journey of learning how to develop wasmCloud applications. 

For example, starting an actor is covered in the creating an actor guide. Starting a provider is covered in the creating a provider guide, etc.

Reply:
Maybe instead of "the following starting actions", we could say that in this guide, we're going to be taking a **tour** of some of the most common activities in the wasmCloud ecosystem, like starting and configuring actors and providers.

I think the "tour" phrase here gives people a better perspective that you're going to walk them through things in this guide that are also discussed at length elsewhere in the docs.

Reply:
I like `tour` too, since we aren't requiring any action by the developer other than copy/pasting the host ID. everything else is stubbed out